techmatters/flex-plugins
# Resource View

Primary reviewer:

## Description
- This PR modifies the 'ViewResource' component and adds functionality to use the updated data model for resources. 
- Mocks are here: https://marvelapp.com/prototype/2i5hhd8g/screen/90702290
![Screenshot 2023-03-17 at 11 47 07 AM](https://user-images.githubusercontent.com/102122005/225955483-d721ad84-1631-4dc2-ab1b-ab7a4a207b97.png)

### Checklist
- [n/a] New tests added
- [n/a] Feature flags added
- [n/a] Strings are localized
- [na] Tested for chat contacts
- [n/a] Tested for call contacts

### Related Issues
Fixes # 1172

### Verification steps
 - Check the first five resources by search
- Compare with the mocks: https://marvelapp.com/prototype/2i5hhd8g/screen/90702290
